recipe/build.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/sh
 mkdir $PREFIX/lib
-cp $RECIPE_DIR/bfg $PREFIX/bin
+echo "#!/bin/sh" >> $PREFIX/bin/bfg
+echo "java -jar ${PREFIX}/lib/bfg.jar \"$@\"" >> $PREFIX/bin/bfg
```

Comment:
This should be changed to

```bash
echo "java -jar ${CONDA_PREFIX}/lib/bfg.jar \"$@\"" >> $PREFIX/bin/bfg
```

Reply:
@conda-forge/core sorry for bothering you, but I'm not sure about the difference between `PREFIX` and `CONDA_PREFIX`, and consequently what to use in this case. It seems from the [docs](https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html#environment-variables-that-affect-the-build-process) that the latter is only relevant for the _build process_, and therefore, in this scenario, we should be using `PREFIX`. Can anyone give us some more insight?

Reply:
Yeah, that's right, basically only use PREFIX, unless you are messing with conda itself

Reply:
Thanks @scopatz!